### PR TITLE
MODX 3.0 Support

### PR DESCRIPTION
**Pull request by crystaldaking**
MODX 3.0 Support

### CODE REVIEW

File: core/components/migx/docs/readme.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
---------------------
+ы--------------------
```
Comment (Jako) [inline core/components/migx/docs/readme.txt]: Whats this?

**Comment by Bruno17** [comment]
Thank you very much, @crystaldaking, for that PR
I think, this should go into a new branch, 'modx3' or something.
This will not be backwards - compatible to revolution 2.x, right?
Not sure, how to handle that.

I've installed your zip - file, which you posted on slack on a new MODX alpha 1
What I've found so far at a very quick look:

- seems to install fine so far
- after Installation MODX Package Manager shows **Not Installed**
- I can create and use simple MIGX - TVs
- MIGX config-CMP shows **Requested processor not found** in its grid

File: core/components/migx/docs/readme.txt (hunk 1 above)
Comment (crystaldaking) [reply core/components/migx/docs/readme.txt]: Sorryy, its my macbook keyboard

**Comment by crystaldaking** [comment]
> I think, this should go into a new branch, 'modx3' or something
This will not be backwards - compatible to revolution 2.x, right?
Not sure, how to handle that.

No way :( 
Yep, we need to create 3.x branch

> after Installation MODX Package Manager shows Not Installed
Hmm

> MIGX config-CMP shows Requested processor not found in its grid
I will add it to my backlog

**Review by Bruno17 — APPROVED** [review]

**Comment by Ibochkarev** [comment]
@Bruno17 Good afternoon!

Tell me, please, can the community expect MODX 3 support?

**Comment by Bruno17** [comment]
I'm currently trying to work on it. But this PR has to much changes at once and is difficult to test. 
I will try to implement and test changes of that PR in small tests. I think, there has more to be done, than just this PR to get everthing working.

**Pull request closed without merging**